tsconfig.json

```diff
@@ -1,5 +1,18 @@
 {
-  "compilerOptions": {},
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
+      "@root/*": [
+        "./*"
+      ],
+      "@src/*": [
+        "./src/*"
+      ],
+      "@components/*": [
+        "./src/components/*"
+      ]
+    }
+  },
```

Comment:
Hmm think we could do this better unless not having webpack is causing us to do this. Ik with webpack we can do it in a cleaner way.

```
baseUrl: "./src",
```
plus probably some other config changes

Reply:
I forgot to clean that up to have a single root. Unfortunately we need to have it in both babel and tsconfig unless you know of a way for webpack to get typescript to resolve properly.

Reply:
Let me look into it a little more

Reply:
This more-or-less stayed the same. I was able to simplify the babel.config.js though so it uses what's in here to populate over there.